app/screens/people-screen/circles/share-circles-image.tsx

```diff
@@ -0,0 +1,243 @@
+import { gql } from "@apollo/client"
+import { useCirclesSharesQuery, WelcomeProfile } from "@app/graphql/generated"
+import { makeStyles, Text, useTheme } from "@rneui/themed"
+import { forwardRef, useMemo, useRef } from "react"
+import { View, Share as NativeShare } from "react-native"
+import Icon from "react-native-vector-icons/Ionicons"
+
+import Logo from "@app/assets/logo/app-logo-dark.svg"
+
+import { captureRef } from "react-native-view-shot"
+import Share from "react-native-share"
+import { Circle } from "@app/components/circle"
+import { LinearGradient } from "react-native-linear-gradient"
+import crashlytics from "@react-native-firebase/crashlytics"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { getInviteLink } from "./helpers"
+
+gql`
+  query CirclesShares {
+    me {
+      username
+      defaultAccount {
+        ... on ConsumerAccount {
+          welcomeProfile {
+            allTimePoints
+            allTimeRank
+            innerCircleAllTimeCount
+            innerCircleThisMonthCount
+            leaderboardName
+            outerCircleAllTimeCount
+            outerCircleThisMonthCount
+            thisMonthPoints
+            thisMonthRank
```

Comment:
we should probably just fetch what we show

Reply:
We are showing everything except for the leaderboardName which I'll remove - hold up!